pkg/rancher-desktop/assets/specs/command-api.yaml

```diff
@@ -238,50 +238,63 @@ components:
           properties:
             adminAccess:
               type: boolean
-              x-rd-platforms: ['darwin', 'linux'] # Only in the specified platforms
+              x-rd-usage: Enable privileged operations
```

Comment:
Missing period at the end, but I think we would do better by dropping the final period for single-sentence descriptions. Makes it less cluttered. And all descriptions should be one sentence or phrase